source/Octo.Tests/Commands/CommandDesignFixture.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using FluentAssertions;
+using NUnit.Framework;
+using Octopus.Cli.Commands;
+using Octopus.Cli.Infrastructure;
+
+namespace Octo.Tests.Commands
+{
+    [TestFixture]
+    public class CommandDesignFixture
+    {
+        [Test]
+        public void ShouldAttachWithCorrectAttribute()
+        {
+            var commandClassNamesWithoutCorrectAttribute = Assembly.GetAssembly(typeof(ICommand)).GetTypes()
+                .Where(t => typeof(ICommand).IsAssignableFrom(t) &&
```

Comment:
I think this is easier to read:
```suggestion
                .Where(t => t.IsAssignableTo<ICommand>() &&
```